subjects/PE/x86/IncorrectUserSignature/IncorrectUserSignature.c

```diff
@@ -8,6 +8,6 @@
 void test6(cdecl_class * c, int32 a, int32 b)
 {
 	c->vtbl->sum(c, a, b);
-	c->vtbl->method04(c, eax_37);
+	c->vtbl->method04(c, <invalid>);
```

Comment:
@ uxmal Is it OK? Should I modify `Coalescer` to avoid coalesce of `Constant.Invalid` expression and emit
```
word32 eax_37;
c->vtbl->method04(c, eax_37);
```